src/components/Register.jsx

```diff
@@ -43,108 +50,132 @@ function Register() {
   const successful = (response) => {
     console.log({ id: response }, "new user added");
     toastr["success"]("register success");
-
   };
 
   const error = (response) => {
     console.log({ error: response }, "registerUser error");
     toastr["error"]("error in registration");
   };
 
-  const cancelRegistration = () => {
-    setUserFormData(initialUserData);
-  }
-
   return (
     <Fragment>
-      <form className="container shadow mb-3">
-        <p></p>
-        <div className="mb-3">
-          <label htmlFor="exampleInputEmail1" className="form-label">
-            Email Address
-          </label>
-          <input
-            value={userFormData.email}
-            onChange={onFormFieldChange}
-            type="email"
-            className="form-control"
-            id="exampleInputEmail"
-            name="email"
-            placeholder="Enter Email"
-          />
+      <div className={`mx-0 px-0 ${styles.headerBackground}`}/>
+      <h1 className={styles.h1}>Register</h1>
+        <div className={styles.checkboxContainer}>
```

Comment:
Since Kristin is doing the organization register page, you can remove the checkbox distinction on the register form.

Reply:
done